### PR TITLE
Enable longoptions for grd2cpt.

### PR DESCRIPTION
**Pull request by rbdavis**
**Description of proposed changes**

Python differences:

  Changed -A transparency (conflicts with common options -t
  transparency) to alpha.

  Changed -H output to stdout.

Julia differences:

  Removed -A transparency alias (conflicts with common options -t
  transparency).

  Changed -S symetric (misspelling) to symmetric.

  Used wrap for -W wrapping directive (w) instead of as an alias
  for the main longoption string.

Fixes #

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

File: src/longopt/grd2cpt_inc.h [hunk 1]
```diff
@@ -25,70 +25,46 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
...
-	{ 0, 'M', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'F', "format|color_model",
+	          "c,g,h,r,R,x",       "cmyk,gray,hsv,rgb,name,hex",
+	          "c,k",               "categorical,key",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
+	{ 0, 'G', "truncate",          "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'H', "stdout",            "", "", "", "", GMT_TP_STANDARD },
```
Comment (joa-quim) [inline src/longopt/grd2cpt_inc.h:46]: `stdout` is not a good name (programmers slang). Specially for wrappers where the concept does not make much sense.
Don't recall why I didn't implement this option in GMT.jl. Maybe `cptfile`?

**Review by joa-quim — APPROVED** [review]

File: src/longopt/grd2cpt_inc.h (hunk 1 above)
Comment (PaulWessel) [reply src/longopt/grd2cpt_inc.h:46]: How about savecpt?

Comment (joa-quim) [reply src/longopt/grd2cpt_inc.h:46]: `savecpt` looks good

**Review by PaulWessel — APPROVED** [review]
Let -H be --savecpt

**Pull request merged**